### ISSUE TITLE
Error [ConfigError]: Config (unnamed): Key "language" on convertErrorToWarningPerFile

### ISSUE THREAD

**Issue by danielpza** [mizdra/eslint-interactive#360]
When choosing the option to convert the error to warning per file it crashes:

```
✔ Which rules would you like to apply action? · @typescript-eslint/no-explicit-any
✔ Which action do you want to do? · convertErrorToWarningPerFile
✔ Leave a code comment with your reason for fixing (Optional) · 
✖ Fixing...
Error [ConfigError]: Config (unnamed): Key "language": Expected string in the form "pluginName/objectName" but found "[object Object]".
    at rethrowConfigError (/home/daniel/projects/typescript-transform-paths/node_modules/@eslint/config-array/dist/cjs/index.cjs:303:8)
    at /home/daniel/projects/typescript-transform-paths/node_modules/@eslint/config-array/dist/cjs/index.cjs:1098:5
    at Array.reduce (<anonymous>)
    at FlatConfigArray.getConfigWithStatus (/home/daniel/projects/typescript-transform-paths/node_modules/@eslint/config-array/dist/cjs/index.cjs:1091:43)
    at FlatConfigArray.getConfig (/home/daniel/projects/typescript-transform-paths/node_modules/@eslint/config-array/dist/cjs/index.cjs:1120:15)
    at Linter._verifyWithFlatConfigArray (/home/daniel/projects/typescript-transform-paths/node_modules/eslint/lib/linter/linter.js:2052:36)
    at Linter.verify (/home/daniel/projects/typescript-transform-paths/node_modules/eslint/lib/linter/linter.js:1547:61)
    at verifyAndFix (/home/daniel/projects/typescript-transform-paths/node_modules/eslint-interactive/src/eslint/linter.ts:53:8)
    at Core.fix (/home/daniel/projects/typescript-transform-paths/node_modules/eslint-interactive/src/core.ts:207:27)
    at SerializableCore.convertErrorToWarningPerFile (/home/daniel/projects/typescript-transform-paths/node_modules/eslint-interactive/src/core-worker.ts:47:18)
```

```
❯ yarn eslint --version
v9.8.0
❯ yarn eslint-interactive --version
11.0.2
```

The other options seem to be working fine so far.

btw thanks for this awesome package. I've used it in some large scale codebases and it has been a huge time saver.

**Comment by mizdra** [comment]
Can you provide a repository that reproduces the problem? Without it, we cannot investigate.

**Comment by danielpza** [comment]
Hey @mizdra, thanks for the quick reponse. 

If it helps although not minimal this is where I got the error https://github.com/LeDDGroup/typescript-transform-paths/compare/fix-eslint

I check again try to make a minimal reproduction

**Comment by tamasfabi** [comment]
I did encounter the same error message on my private pnpm monorepo too (with flatconfig for eslint), when using with these packages:

```
"@eslint/compat": "^1.1.1",
"@eslint/js": "^9.8.0",
"@typescript-eslint/parser": "^8.0.1",
"eslint": "^9.8.0",
"eslint-config-prettier": "^9.1.0",
"eslint-interactive": "^11.0.2",
"eslint-plugin-jsx-a11y": "^6.9.0",
"eslint-plugin-react": "^7.35.0",
"eslint-plugin-react-hooks": "^4.6.2",
"eslint-plugin-simple-import-sort": "^12.1.1",
```

**Comment by mizdra** [comment]
@danielpza Thanks for providing your repository! I could reproduce the issue. I will fix it.

**Comment by mizdra** [comment]
This problem seems to occur with ESLint 9.5.0+. It is not reproduced in 9.4.0 or lower.

minimum reproduction:
https://github.com/mizdra/typescript-transform-paths/tree/minimum-reproduction

**Comment by danielpza** [comment]
Sorry, I couldn't provide a minimal repo, I got distracted with work.

Thanks for taking a look @mizdra :heart:.

**Comment by mizdra** [comment]
@danielpza, @tamasfabi This bug fix is shipped in v11.0.3. Thanks for contributing!

- https://github.com/mizdra/eslint-interactive/releases/tag/v11.0.3

**Comment by nocive** [comment]
Can also confirm that it's fixed, thanks :heart:
